signer/src/error.rs

```diff
@@ -442,6 +442,10 @@ pub enum Error {
     #[error("stacks transaction rejected: {0}")]
     StacksTxRejection(#[from] crate::stacks::api::TxRejection),
 
+    /// The stacks fee was too high.
+    #[error("coordinator Stacks txn with fee too high: {0}. Highest acceptable fee: {1}")]
+    TooHighStacksTxFee(u64, u64),
```

Comment:
Suggestion: I'd lead with `Stacks*` since it aligns with most others. Like, `StacksFeeLimitExceeded` or something.